PowerSync/build.gradle.kts

```diff
@@ -35,6 +36,14 @@ kotlin {
     }
 }
 
+skie {
+    features {
+        group {
+            SuspendInterop.Enabled(false) // or false
```

Comment:
Nitpick: The comment doesn't make sense here, maybe it can explain why we are disabling suspend for SKIE.

Reply:
Sorry I missed that. I've removed the comment and replaced it with an explanation.

Reply:
Another little note: I think `compiling` is a better word to use here rather than `implementing`.

Reply:
Oh shucks sorry missed this before I merged. Will include it in the next change I make

Reply:
Oh no wait there isn't an issue at compile time. The issue happens when you implement parts of the SDK in Swift.